badgr-fullstack-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,282 @@
+      }
+    },
+	"badgrSuperUsername": {
+      "type": "string",
+      "defaultValue": "badgruser",
+      "metadata": {
+        "description": "Badgr Application super username."
+      }
+    },
+    "badgrSuperUserPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Badgr Application super user password."
+      }
+    },
+    "dnsLabelPrefix": {
+      "type": "string",
+      "metadata": {
+        "description": "DNS Label for the Public IP. Must be lowercase. It should match with the following regular expression: ^[a-z][a-z0-9-]{1,61}[a-z0-9]$ or it will raise an error."
+      }
+    },
+    "vmSize": {
+      "type": "string",
+      "defaultValue": "Standard_D3_v2",
+      "allowedValues": [
+        "Standard_A2",
+        "Standard_A3",
+        "Standard_A4",
+        "Standard_A5",
+        "Standard_A6",
+        "Standard_A7",
+        "Standard_A8",
+        "Standard_A9",
+        "Standard_A10",
+        "Standard_A11",
+        "Standard_D2",
+        "Standard_D3",
+        "Standard_D4",
+        "Standard_D11",
+        "Standard_D12",
+        "Standard_D13",
+        "Standard_D14",
+        "Standard_D2_v2",
+        "Standard_D3_v2",
+        "Standard_D4_v2",
+        "Standard_D5_v2",
+        "Standard_D11_v2",
+        "Standard_D12_v2",
+        "Standard_D13_v2",
+        "Standard_D14_v2",
+        "Standard_D15_v2"
+      ],
+      "metadata": {
+        "description": "Virtual machine size."
+      }
+    }
+  },
+  "variables": {
+    "apiVersion": "2015-06-15",
+    "scriptDownloadUri": "https://raw.githubusercontent.com/satyarapelly/azure-quickstart-templates/master/badgr-fullstack-ubuntu/",
+    "installScript": "install-badgr.sh",
+    "installCommand": "[concat('bash -c ''nohup ./', variables('installScript'), ' -u ', parameters('badgrSuperUsername'), ' -p ', parameters('badgrSuperUserPassword'), ' -e ', parameters('badgrSuperUserEmail'),' </dev/null &>/var/log/azure/badgr-install.log &''')]",
+    "vmName": "badgrvm",
+    "osImagePublisher": "Canonical",
+    "osImageOffer": "UbuntuServer",
+    "osImageSKU": "16.04-LTS",
+    "publicIPAddressName": "myPublicIP",
+    "publicIPAddressType": "Dynamic",
+    "OSDiskName": "osdisk",
+    "nicName": "vmNic",
+    "virtualNetworkName": "VNET",
+    "vnetID": "[resourceId('Microsoft.Network/virtualNetworks',variables('virtualNetworkName'))]",
+    "subnetRef": "[concat(variables('vnetID'),'/subnets/',variables('subnetName'))]",
+    "subnetName": "Subnet",
+    "addressPrefix": "10.0.0.0/16",
+    "subnetPrefix": "10.0.0.0/24",
+    "nsgName": "node-nsg",
+    "nsgID": "[resourceId('Microsoft.Network/networkSecurityGroups',variables('nsgName'))]",
+    "storageAccountType": "Standard_LRS",
+    "storageAccountName": "[concat(uniquestring(resourceGroup().id), 'vhdsa')]",
+    "vhdBlobContainer": "vhds"
+  },
+  "resources": [
+    {
+      "apiVersion": "[variables('apiVersion')]",
```

Comment:
I removed all variable declaration and used ApiVersion value wherever is used. thanks!